infra/Pos/Arbitrary/Slotting.hs

```diff
@@ -6,13 +6,12 @@ import           Test.QuickCheck                   (Arbitrary (..))
 import           Test.QuickCheck.Arbitrary.Generic (genericArbitrary, genericShrink)
 
 import           Pos.Arbitrary.Core                ()
-import           Pos.Slotting.Types                (EpochSlottingData (..),
-                                                    SlottingData (..))
+import           Pos.Slotting.Types                (EpochSlottingData (..), SlottingData)
 
 instance Arbitrary EpochSlottingData where
     arbitrary = genericArbitrary
     shrink = genericShrink
 
 instance Arbitrary SlottingData where
     arbitrary = genericArbitrary
-    shrink = genericShrink
+    shrink = genericShrink    
```

Comment:
It's recommended to setup your editor to remove trailing whitespaces automatically.